packages/react-components/src/link/src/Link.css

```diff
@@ -180,62 +189,63 @@
 
 /* VARIATIONS */
 /* PRIMARY */
-.o-ui-link-primary {
-    color: var(--primary-500);
+.o-ui-link.o-ui-link-primary {
```

Comment:
I don't think you need `.o-ui-link` here

Reply:
it was needed for specificity reasons
```
.o-ui-link {
...    color: var(--primary-text);
...
}
```

since link has a color set we need more specificity.

Reply:
Since we now set the marine900 at the themeprovider level this won't be necessary.